Edge/FIX_PERMISSIONS.cmd

```diff
@@ -0,0 +1,3 @@
+REM Grant necessary fs permissions to the Edge sandbox user to allow access to the extension's directory
+cd /d %~dp0
+icacls "." /grant "*S-1-15-2-3624051433-2125758914-1423191267-1740899205-1073925389-3782572162-737981194":"(OI)(CI)(WDAC,WO,GE)" 
```

Comment:
This isn't in any of the Edge documentation... can someone confirm that's its necessary for XKit?